src/openforms/scss/admin/_app_overrides.scss

```diff
@@ -98,6 +98,31 @@ $djai-padding--tablet: 30px;
   padding-top: 1em;
 }
 
+.admin-login-divider {
+  text-align: center;
+  clear: both;
+  padding-top: 1em;
+
+  width: 100%;
+  text-align: center;
+  border-bottom: 1px solid #ccc;
+  line-height: 0.1em;
+  margin: 10px 0 10px;
+
+  span {
+    background: #fff;
+    padding: 0 10px;
+  }
+}
```

Comment:
This doesn't work in dark mode:

![image](https://github.com/open-formulieren/open-forms/assets/5518550/9a48daeb-1c5e-48b1-984b-949d8476d21c)

You can basically never use literal values for colors anymore, but should always point to the relevant CSS variable. See `django/contrib/admin/static/admin/css/base.css` for a reference of built in django vars, and our own stylesheets define some extra variables you can use.